src/test/kontrol/TermRepoCollateralManager.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity 0.8.23;
+
+import "forge-std/Test.sol";
+import "kontrol-cheatcodes/KontrolCheats.sol";
+
+import "src/interfaces/term/ITermRepoCollateralManager.sol";
+
+contract TermRepoCollateralManager is ITermRepoCollateralManager, Test, KontrolCheats {
+    mapping(address => uint256) _maintenanceCollateralRatios;
+    address[] _collateralTokens;
+
+    function initializeSymbolic() public {
+        kevm.symbolicStorage(address(this));
+
+        // For simplicity, choose an arbitrary number of collateral tokens
+        vm.assume(_collateralTokens.length == 3);
```

Comment:
For simplicity, we use a fixed number of collateral tokens. We can change this to a different number if desired, and it wouldn't affect the complexity of the formal verification. But if we used a symbolic value here instead, that would considerably increase the number of branches in the symbolic execution without providing any significant extra coverage. So I don't think it's worth it.